client/src/Components/Common/NavBar/NavBar.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react"
+
+import "./NavBar.css"
+import AvailableRooms from "../AvailableRooms/AvailableRooms"
+
+class NavBar extends Component {
+  state = {
+    navBarStatus : true
+  }
+
+  handleEvent = () => {
```

Comment:
Great! try to use one method to toggle the boolean state rather than two.
preState will help you